samples/features/security/always-encrypted-with-secure-enclaves/sample-application/README.md

```diff
@@ -0,0 +1,17 @@
+This is a sample C# client app used to demo the coding aspects of Always Encrypted with VBS enclaves using the ContosoHR Database. 
```

Comment:
Can you please add sample command-line examples that show how to use / set the environment variables? This will make it easier for someone to build the code and execute without having to go through the whole logic.

Reply:
Done